pages/help/help_contact.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Contact us
+permalink: /help/contact/
+layout: layouts/info-page
+searchbox: true
+excerpt: Contact the .gov team with questions or comments. 
+tags: help
+eleventyNavigation:
+  key: help
+  order: 4 
+---
+
+## Contact us
```

Comment:
No need to this header because the template will put in a H1 header with the title